src/DynamoCore/Core/PulseMaker.cs

```diff
@@ -110,11 +115,11 @@ private void OnTimerTicked(object state)
 
                 // Further evaluation was requested.
                 evaluationRequestPending = false;
-                BeginRunExpression();
+                BeginRun();
             }
         }
 
-        private void BeginRunExpression()
+        private void BeginRun()
         {
```

Comment:
This is just unnecessarily wordy, so I changed it.  Technically a Dynamo graph isn't really an expression anyways.